### PR TITLE
Symfony 3.0 support

### PR DESCRIPTION
**Pull request by Ener-Getick**
This PR adds symfony 3.0 support.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,20 +12,26 @@
             "homepage": "https://sonata-project.org/"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/Ener-Getick/PHP-CS-Fixer"
+        }
+    ],
```
Comment (Ener-Getick) [inline composer.json]: Must be removed once https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1510 is merged.

**Comment by OskarStark** [comment]
Thank you @Ener-Getick 

can you please check, why travis failed?

ping @Soullivaneuh

**Comment by Ener-Getick** [comment]
I think that this is related to the constraint of ``fabpot/php-cs-fixer`` and that it will be fixed when I will revert it.

**Comment by rande** [comment]
It will be better to not have this dependency inside composer.json file ... @Soullivaneuh will be happy ;)

File: .php_cs [hunk 2]
```diff
@@ -5,7 +5,6 @@ $finder = Symfony\CS\Finder\DefaultFinder::create()
 ;
 
 return Symfony\CS\Config\Config::create()
-    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
```
Comment (Soullivaneuh) [inline .php_cs]: Why this?

File: .travis.yml [hunk 3]
```diff
@@ -35,19 +35,18 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.8.*@dev
     - php: 5.6
-      env: SYMFONY_VERSION="3.0.x-dev as 2.8"
+      env: SYMFONY_VERSION="3.0.x-dev" DEPENDENCIES="dev" COMPOSER_FLAGS="--prefer-stable"
```
Comment (Soullivaneuh) [inline .travis.yml]: Just use `3.0.*@dev`. The dev tag could be removed when SF 3.0 will be stable.

**Comment by Soullivaneuh** [comment]
> It will be better to not have this dependency inside composer.json file ... @Soullivaneuh will be happy ;)

Are you talking about VCS? Yes, but @Ener-Getick plan to do it. Will merge this PR after that.

File: .php_cs (hunk 2 above)
Comment (Ener-Getick) [inline .php_cs]: Because this function doesn't exist in Php-cs-fixer 0.* which is needed to test sf 2.3.

Comment (Ener-Getick) [inline .php_cs]: And because this is the default value.

Comment (Soullivaneuh) [inline .php_cs]: Yeah ok. This is why I want to completely remove this dependencies and use StyleCI.

See here: https://github.com/sonata-project/SonataAdminBundle/pull/3273

Comment (Ener-Getick) [inline .php_cs]: Another solution would be to use the same solution used in Php-cs-fixer (a regex replacement).

File: .travis.yml [hunk 4]
```diff
@@ -27,27 +27,21 @@ matrix:
     - php: 5.3
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: SYMFONY_VERSION=2.3.*
```
Comment (Ener-Getick) [inline .travis.yml]: I removed this test as symfony 2.3 is already tested in the first one.

**Comment by Ener-Getick** [comment]
Php-cs-fixer now supports symfony 3.0. 
This PR is ready to be merged if you have nothing else to change.

**Comment by rande** [comment]
@Ener-Getick Thanks

**Comment by OskarStark** [comment]
:+1:

**Pull request merged**